manifests/oracle-cluster/prod/helium/mobile-oracle.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: rds-mobile-oracle-user-access
       containers:
         - name: mobile-oracle
-          image: public.ecr.aws/j4f9f0n7/distributor-oracle:0.0.34
+          image: public.ecr.aws/j4f9f0n7/distributor-oracle:0.0.35
```

Comment:
This version bump removes the divide by 100 of rewards, since the current oracle writes 8 decimals